### PR TITLE
created instructor dashboard

### PR DESCRIPTION
**Pull request by Akshaya754**

### CODE REVIEW

File: client/src/assets/style.css [hunk 1]
```diff
@@ -56,6 +56,11 @@
     background-repeat: no-repeat;
 }
 
+#instructor-dashboard{
+ background: linear-gradient(45deg, rgba(255, 255, 255, 0.75), rgba(167, 195, 222, 0.42));
+ font-family: 'Rubik', sans-serif;
+}
```
Comment (mishelshaji) [inline client/src/assets/style.css]: Format the code.

File: client/src/components/instructor/Header.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+				<img
+					src={Logo}
+					width="200"
+					className="d-inline-block align-top"
+					alt="myLearn"
+					id="logo"
+				/>
+			</Navbar.Brand>
+		</Navbar>
+	);
```
Comment (mishelshaji) [inline client/src/components/instructor/Header.js]: Format the code.

File: client/src/components/instructor/Header.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import Navbar from "react-bootstrap/Navbar";
+import { Button } from 'react-bootstrap';
+import Logo from "../../images/logo.png";
+import { BsList } from 'react-icons/bs';
+
```
Comment (mishelshaji) [inline client/src/components/instructor/Header.js:5]: Remove the extra blank line.

File: client/src/pages/instructor/InstructorDashboard.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+								<td>2022-02-01</td>
+								<td>2022-02-10</td>
+							</tr>
+						</tbody>
+					</Table>
+				</div>
+			</Col>
+		</>
+	);
+}
```
Comment (mishelshaji) [inline client/src/pages/instructor/InstructorDashboard.js:128]: What is this? Format the code.

File: client/src/pages/instructor/InstructorLayout.js [hunk 5]
```diff
@@ -1,10 +1,23 @@
...
     <>
-      <Outlet></Outlet>
+      <Header showSidebar={showSidebar} setShowSidebar={setShowSidebar} />
+      <Container fluid>
+        <Row id='instructor-dashboard'>
+          <SideNavigationBar showSidebar={showSidebar} setShowSidebar={setShowSidebar} />
+          <Outlet></Outlet>
+        </Row>
+      </Container>
+      <Footer />
```
Comment (mishelshaji) [inline client/src/pages/instructor/InstructorLayout.js]: Before you make the next commit, please read about js and React coding conventions and format the code.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/src/assets/style.css (hunk 1 above)
Comment (Akshaya754) [reply client/src/assets/style.css]: code formatted

File: client/src/components/instructor/Header.js (hunk 2 above)
Comment (Akshaya754) [reply client/src/components/instructor/Header.js]: code formatted

File: client/src/components/instructor/Header.js (hunk 3 above)
Comment (Akshaya754) [reply client/src/components/instructor/Header.js:5]: Blank space removed.

File: client/src/pages/instructor/InstructorDashboard.js (hunk 4 above)
Comment (Akshaya754) [reply client/src/pages/instructor/InstructorDashboard.js:128]: code formatted

File: client/src/pages/instructor/InstructorLayout.js (hunk 5 above)
Comment (Akshaya754) [reply client/src/pages/instructor/InstructorLayout.js]: code formatted

**Review by mishelshaji — APPROVED** [review]

File: client/src/App.js [hunk 6]
```diff
@@ -14,7 +14,8 @@ import StudentRegistration from './pages/student/StudentRegistration';
 import InstructorRegistration from './pages/instructor/InstructorRegistration';
 import EnrolledCourses from './pages/student/EnrolledCourses';
 import Order from './pages/student/Order';
-import StudentProfile from './pages/student/StudentProfile';
+import InstructorDashboard from './pages/instructor/InstructorDashboard';
+import InstructorCourses from './pages/instructor/InstructorCourses.js';
```
Comment (mishelshaji) [inline client/src/App.js]: Format the routes and group child routes.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

**Review by mishelshaji — APPROVED** [review]

**Review by SreenathES — APPROVED** [review]

**Review by SreenathES — APPROVED** [review]

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**